src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -349,6 +349,34 @@ public string License
             }
         }
 
+        private string copyrightHolder;
+        /// <summary>
+        /// The name of the author who holds this package's copyright.
+        /// </summary>
+        public string CopyrightHolder
```

Comment:
@QilongTang  @OliverEGreen  do these new properties make it up to the published package somehow?

Reply:
ahh, I see @OliverEGreen 's comment now that more pluming is required to actually use these.